Sources/Angle.swift

```diff
@@ -0,0 +1,136 @@
+//
+//  Angle.swift
+//  Euclid
+//
+//  Created by Ioannis Kaliakatsos on 22.11.2020.
+//  Copyright © 2020 Nick Lockwood. All rights reserved.
+//
+
+import Foundation
```

Comment:
If you branch from develop instead of Master then these changes should already be included.

Reply:
I did, at least that was the intention :)